worker/buildbot_worker/base.py

```diff
@@ -320,6 +326,20 @@ def remote_getWorkerInfo(self):
                 if os.path.isfile(filename):
                     with open(filename, "r") as fin:
                         files[f] = fin.read()
+
+        if os.path.exists(self.os_release_file):
+            with open(self.os_release_file, "r") as fin:
```

Comment:
you should open with 'rb' if you want to manage bytes

Reply:
I don't think I want to manage bytes.